doc/autoinstallation.md

```diff
@@ -0,0 +1,94 @@
+## Introduction
+
+The workflow of the autoinstallation is customized via the [control
+file](https://github.com/yast/yast-installation/blob/master/doc/control-file.md) 
+like it is in the installation mode but the principal steps are not exactly
+the same, and also while in a normal installation the [Second
+Stage](https://www.suse.com/documentation/sles-12/singlehtml/book_autoyast/book_autoyast.html#overviewandconcept)
+is not needed, in autoinstallation the installation proccess is divided in
+two and in the _Second Stage_ is where and when the system configuration is
+really done.
+
+There are two ways to give a profile to _AutoYaST_, with (`autoyast` or with
+`autoyast2` parameters), the principal difference from networking point of 
+view is that `autoyast` will not fetch the profile which means that doesn't
```

Comment:
"`autoyast` will not fetch the profile" confusingly suggests that the profile is ignored. How about this:

> [the difference is that] `autoyast` leaves the fetching of the profile to YaST which means Linuxrc does not need to configure the network, while for `autoyast2` Linuxrc fetches the profile and may need need to configure the network.